ui/client/reducers/selectors/settings.ts

```diff
@@ -1,9 +1,10 @@
 import {RootState} from "../index"
 import {SettingsState} from "../settings"
 import {createSelector} from "reselect"
+import {ProcessDefinitionData} from "../../types"
 
 const getSettings = (state: RootState): SettingsState => state.settings
 
 export const getFeatureSettings = createSelector(getSettings, s => s.featuresSettings)
 export const getLoggedUser = createSelector(getSettings, s => s.loggedUser)
-export const getProcessDefinitionData = createSelector(getSettings, s => s.processDefinitionData || {})
+export const getProcessDefinitionData = createSelector(getSettings, s => s.processDefinitionData || {} as ProcessDefinitionData)
```

Comment:
How does work this `as ..`?

Reply:
https://www.typescriptlang.org/docs/handbook/basic-types.html#type-assertions

Reply:
O nice! Thanks ;)